api/src/certification/session-management/application/jobs/cpf-export-sender-job-controller.js

```diff
@@ -0,0 +1,25 @@
+import { mailService } from '../../../../../lib/domain/services/mail-service.js';
+import { JobScheduleController } from '../../../../shared/application/jobs/job-schedule-controller.js';
+import { config } from '../../../../shared/config.js';
+import { logger } from '../../../../shared/infrastructure/utils/logger.js';
+import { usecases } from '../../domain/usecases/index.js';
+
+const { cpf } = config;
+
+class CpfExportSenderJobController extends JobScheduleController {
```

Comment:
Suggestion : Renommer la classe et le fichier pour inclure le terme schedule ?
Ca permettrait de différencier, dans le nommage, les cron des jobs classiques 😄

```suggestion
class CpfExportSenderJobScheduleController extends JobScheduleController {
```